docs/modules/ROOT/pages/cli/getting_started_in_cli.adoc

```diff
@@ -0,0 +1,168 @@
+
+== Getting started in the CLI
```

Comment:
I thought we decided it would be `Get started` and your other PR referred to it as `Get started`: https://github.com/redhat-appstudio/docs.stonesoup.io/pull/83/files